products/amazon-neptune.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Amazon-Neptune
+category: db
```

Comment:
We prefer using `service` for hosted services, and `db` for actual databases you can install and run on your own.

```suggestion
category: service
```